### PR TITLE
Tadpole warning

### PR DESCRIPTION
**Pull request by Lefunny**
## About The Pull Request

A tad computer warn for when you click on a broken tad computer

## Why It's Good For The Game

Now you know when the AI broke the tad or a xeno.

## Changelog
:cl:
add: Added new mechanics or gameplay changes
Now the tadcomputer have a text for when marines click on it and its broken

### CODE REVIEW

File: code/modules/shuttle/mini_dropship.dm [hunk 1]
```diff
@@ -173,6 +173,10 @@
 	open_prompt = FALSE
 	clean_ui_user()
 
+/obj/machinery/computer/camera_advanced/shuttle_docker/minidropship/attack_hand(mob/user)
+	if(damaged == TRUE)
```
Comment (BraveMole) [inline code/modules/shuttle/mini_dropship.dm:177]:
```suggestion
	if(damaged)
```

File: code/modules/shuttle/mini_dropship.dm [hunk 2]
```diff
@@ -173,6 +173,10 @@
...
+		to_chat(user, "The computer blink and let some weirds sounds, its broken!")
```
Comment (BraveMole) [inline code/modules/shuttle/mini_dropship.dm:178]: needs a span_warning(

Comment (BraveMole) [reply code/modules/shuttle/mini_dropship.dm:178]: and a return

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/shuttle/mini_dropship.dm (hunk 1 above)
Comment (Lefunny) [reply code/modules/shuttle/mini_dropship.dm:177]: oh a friend warned me about this but i forgot to chang, thanks

File: code/modules/shuttle/mini_dropship.dm (hunk 2 above)
Comment (Lefunny) [reply code/modules/shuttle/mini_dropship.dm:178]: ok going to do it

Comment (Lefunny) [reply code/modules/shuttle/mini_dropship.dm:178]: can you say if it worked? i tryed to push it but this is my first time doing it

File: code/modules/shuttle/mini_dropship.dm [hunk 3]
```diff
@@ -173,6 +173,11 @@
 	open_prompt = FALSE
 	clean_ui_user()
 
+/obj/machinery/computer/camera_advanced/shuttle_docker/minidropship/attack_hand(mob/user)
```
Comment (BraveMole) [inline code/modules/shuttle/mini_dropship.dm:176]: i haven't noticed before, but you are declaring a new proc inside another one? what is this?

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/shuttle/mini_dropship.dm (hunk 3 above)
Comment (BraveMole) [reply code/modules/shuttle/mini_dropship.dm:176]: it's in the middle of another proc...

**Pull request closed without merging**